sass/_FacetSearch.scss

```diff
@@ -39,6 +43,12 @@
   .coveo-facet-value-label {
     padding-right: 15px;
   }
+  .CoveoFacet & {
+    margin-top: -1px;
+  }
```

Comment:
Without `PopupUtils`, search results move down a pixel. It happens to look a little better when it's moved down a pixel, but for the sake of keeping the same style, this `margin-top` moves it up a pixel. Unfortunately, introducing a `top` would be absolute.